app/femr/ui/views/history/indexEncounter.scala.html

```diff
@@ -177,15 +177,32 @@
                             <span class="value">@outputStringOrNA(viewModelPharmacy.getProblems.get(problemIterator - 1))</span></p>
                     }
 
-                    @for(prescription <- viewModelPharmacy.getPrescriptions){
-                        <p>
-                            <span class="infoLabel">Dispensed Medication:</span>
-                            @prescription.getAmount @outputStringOrNA(prescription.getName) (@outputStringOrNA(prescription.getMedicationForm).toString().trim)
-                            @for(activeDrug <- prescription.getMedicationActiveDrugs) {
-                                <span class='activeDrug'> @activeDrug.getName @activeDrug.getValue@activeDrug.getUnit </span>
-                            }
+                    @* AJ Saclayan Prescribed Medications *@
+                    @if(viewModelPharmacy.getOriginalMedications.size != null){
+                       <p> <span class="infoLabel">Originally Prescribed Medication:</span></p>
+                        @for(x <- 1 to viewModelPharmacy.getOriginalMedications.size -1) {
+@*//                           defining(viewModelPharmacy.getOriginalMedications.get(x - 1)) { medication =>*@
 
-                        </p>
+                            <p> <i>@viewModelPharmacy.getOriginalMedications.get(x-1).getName</i></p>
+
+                            <p> Replaced with: </p>
+                            <p>@viewModelPharmacy.getOriginalMedications.get(x-1 +1).getName</p>
```

Comment:
So adding just originalId into PrescriptionItem ( of course reflecting the changes where this needs to be used as well)?